CMS-Enterprise/mint-app
# [NOREF] Addressing some weird styles introduced in recent merge conflict

# NOREF

## Description
There were some weirdness introduced to the External Link component from a recent merge conflict. This PR is to address said weirdness. Additionally, some code refactor in the `KeyResource` component in the 2 and 6 pager help article

### BEFORE
![image](https://github.com/user-attachments/assets/8a858fde-8f9f-44f9-a93a-3c45034313be)

### AFTER
![image](https://github.com/user-attachments/assets/f0c48a76-1524-4816-b49d-9af14f24e975)
![image](https://github.com/user-attachments/assets/d50fdb87-5ff5-4e2c-b886-d4c2b28c70e7)
![image](https://github.com/user-attachments/assets/59c167dc-233c-43fe-a741-0daeefc8da09)

## How to test this change

1.  Checkout branch and look through app to ensure things match figma
2. Check the key resource component matches figma in the 2 and 6 pager help article


## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
